pkg/acsengine/pki.go

```diff
@@ -68,6 +72,18 @@ func CreatePki(extraFQDNs []string, extraIPs []net.IP, clusterDomain string, caP
 
 	go func() {
 		var err error
```

Comment:
@wangtt03 thanks for this PR and sorry we've taken awhile to respond in kind. I'd like to hold on this PR and instead focus on automatically creating and deploying TLS certificates when bootstrapping the cluster. This follows the standard feature of other notable Kubernetes provisioning tools like kops, kubeadm, and tack. Creating and managing TLS is hard, and acs-engine should provide this operational detail out of the box. This will satisfy the majority of use cases. We can revisit demand for configuring custom TLS certs after a later date.